app/services/log_event_searcher.rb

```diff
@@ -28,7 +28,7 @@ def search
     result = result.merge(filter_date) if start_date || end_date
     result = result.merge(filter_event) if events.present?
     result = result.merge(filter_query) if query.present?
-    result
+    result.reverse_chronological
```

Comment:
This ordering is now happening in the controller from #2257. It's also likely the cause of the spec failures.

```suggestion
    result
```